src/logics/compound-v3/tokens/index.ts

```diff
@@ -9,3 +12,15 @@ export const mainnetTokens = common.toTokenMap<MainnetTokenSymbols>(mainnetToken
 type PolygonTokenSymbols = keyof typeof polygonTokensJSON;
 
 export const polygonTokens = common.toTokenMap<PolygonTokenSymbols>(polygonTokensJSON);
+
+type BaseTokenSymbols = keyof typeof baseTokensJSON;
+
+export const baseTokens = common.toTokenMap<BaseTokenSymbols>(baseTokensJSON);
+
+type ArbitrumTokenSymbols = keyof typeof arbitrumTokensJSON;
+
+export const arbitrumTokens = common.toTokenMap<ArbitrumTokenSymbols>(arbitrumTokensJSON);
+
+type OptimismTokenSymbols = keyof typeof optimismTokensJSON;
+
+export const optimismTokens = common.toTokenMap<OptimismTokenSymbols>(optimismTokensJSON);
```

Comment:
Please arrange them in chain id (ascending)

Reply:
arranged